Agent key rotation: Recommend grepping for the pubkey (#12758)
DataDog/documentation: content/en/agent/faq/linux-agent-2022-key-rotation.md

gpg-pubkey-fd4bf915-5f573efe
```

> Otherwise, the command returns a non-0 exit code and has no output.

Reviewer: ```suggestion
Otherwise, the command returns a non-0 exit code with no output.
```

> Otherwise, the command returns a non-0 exit code and prints nothing.

Author: ```suggestion
Otherwise, the command returns a non-0 exit code and has no output.
```
